src/dotnetTemplate/Microsoft.PowerShell.Standard.Module.Template/Microsoft.PowerShell.Standard.Module.Template/.template.config/template.json

```diff
@@ -16,18 +16,18 @@
     "PowerShellStandardVersion":{
       "type": "parameter",
       "datatype":"choice",
-      "defaultValue": "5.1.0",
+      "defaultValue": "7.0.0-preview.1",
       "choices": [
         {
-          "choice": "5.1.0",
-          "description": "PowerShell Standard 5.1"
+          "choice": "7.0.0-preview.1",
```

Comment:
This should be an additional choice. Not one that replaces 5.1

Reply:
I agree, but then https://github.com/PowerShell/PowerShellStandard/commit/c7c7ad3fa406631c57a307704bd86c6ff80c676f must also be corrected. This merge would at least unblock template usage.